interfaces/cython/cantera/thermo.pyx

```diff
@@ -340,8 +340,8 @@ cdef class ThermoPhase(_SolutionBase):
             return 1.0
 
     def equilibrate(self, XY, solver='auto', double rtol=1e-9,
-                    int maxsteps=1000, int maxiter=100, int estimate_equil=0,
-                    int loglevel=0):
+                    int max_steps=1000, int max_iter=100, int estimate_equil=0,
+                    int log_level=0, **kwargs):
```

Comment:
The docstring needs to be updated to use the new parameter names, e.g. `:param max_steps:`.